app/animals.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = 0 if health <= 0 else health
+        self.hidden = False
+
+        if self.health > 0:
+            self.add_born(self)
+
+    def __repr__(self) -> str:
+        return "{" + f"Name: {self.name}, Health: {self.health}, " \
+               f"Hidden: {self.hidden}" + "}"
+
+    @staticmethod
+    def add_born(animal: Animal) -> None:
+        Animal.alive.append(animal)
+
+    @staticmethod
+    def remove_dead(animal: Animal) -> None:
+        for index in range(len(Animal.alive)):
+            if animal is Animal.alive[index]:
+                Animal.alive.remove(animal)

```

Comment:
```suggestion
      Animal.alive.remove(animal)

```

Reply:
You can call this method without using a cycle

Reply:
Yes, you are right, I changed the function, but the code was not optimized.
Cycle removed